<?xml version="1.0"?>
<launch>

<node pkg="ia_slam" type="ia_slam_node" name="ia_slam">
  <param name="map_frame"  value="map"/>
  <param name="robot_frame"  value="robot"/>
  <param name="beacon_topic"  value="beacon_echo"/>
  <param name="internal_topic"  value="robot_data"/>
  <param name="sensorPrecision"  value="0.2"/>
  <param name="ros_rate"  value="30"/>
  <param name="nb_outliers"  value="3"/>
  <param name="max_box"  value="10"/>
  <param name="gps_precision"  value="1"/>
</node>

</launch>
